dadi/lib/handlers/image.js

```diff
@@ -920,7 +920,10 @@ ImageHandler.prototype.sanitiseOptions = function (options) {
       if (options[key] !== '0' || settings[0].allowZero || settings[0].default) {
         if (options[key] !== '0' || settings[0].allowZero) {
           if (settings[0].lowercase) value = value.toLowerCase()
-          imageOptions[settings[0].name] = _.isFinite(value) ? parseFloat(value) : value
+          value = _.isFinite(value) ? parseFloat(value) : value
+          if (settings[0].minimumValue && value < settings[0].minimumValue) value = settings[0].minimumValue
```

Comment:
I find these two `if`/`else if` blocks a bit hard to read. Could we perhaps change it to:

```js
if (settings[0].minimumValue && value < settings[0].minimumValue) {
  value = settings[0].minimumValue
} else if (settings[0].maximumValue && value > settings[0].maximumValue) {
  value = settings[0].maximumValue
}
```

Reply:
@adamkdean over to you

Reply:
Sorry @eduardoboucas – I don't usually create blocks for single expressions. Fixed though in 33cf07a. @jimlambie 🚀 ?